src/pages/Exercise1.tsx

```diff
@@ -82,16 +100,33 @@ const Exercise1: FC = () => {
                       { id: 3, name: '22', displayName: '22' },
                       { id: 4, name: '23', displayName: '23' },
                     ]}
+                    correctAnswer={correctAnswer}
+                    setCorrectAnswer={setCorrectAnswer}
+                    index={2}
                   />
                 </span>
               </p>
             </Question>
             <Question
-              correct="Correct! Now Pipi needs to..."
-              incorrect="Not quite. Think about ... Please try again."
+              correct="Correct! Pipi needs to move from address 11 to address 22. Thanks to you, Pipi found all of the boxes correctly!"
```

Comment:
Pipi starts at address 11, so 22-11 = 11 addresses. The correctAnswer is wrong here